app/views/schools/choose_programme/success.html.erb

```diff
@@ -1,96 +1,27 @@
-
-        <%- elsif school_cohort.no_early_career_teachers? -%>
-
-            <h2 class="govuk-heading-m">What to do next</h2>
-            <p class="govuk-body">Your school will not receive any more messages about statutory inductions for ECTs until the next academic year.</p>
-            <p class="govuk-body">If any ECTs do join your school this year, you need to use this service to let us know.</a></p>
-
-        <%- elsif school_cohort.school_funded_fip? -%>
-
-            <h2 class="govuk-heading-m">What to do next</h2>
-            <p class="govuk-body">You need to <a class="govuk-link" href="https://www.gov.uk/government/publications/early-career-framework-reforms-overview/early-career-framework-reforms-overview#FPL"> make your own arrangements with a training provider</a>, approved by the DfE.</p>
-            <p class="govuk-body">You do not need to add information about your ECTs and mentors to this service.</p>
-            <p class="govuk-body govuk-!-margin-bottom-7">If you change your mind about how you want to run your training, <%= render MailToSupportComponent.new("contact us") %></p>
-
-            <p class="govuk-link">
-                <%= govuk_link_to("Continue to manage your training", schools_dashboard_path)  %>
-            </p>
-
-        <%- else -%>
-
-            <ul class="govuk-list govuk-list--bullet">
-                <li>you do not need to use this service to set up your training programme</li>
-                <li>you’ll need to make your own arrangements with a
-                    <a href="https://www.gov.uk/government/publications/early-career-framework-reforms-overview/early-career-framework-reforms-overview#FPL"
-                    class="govuk-link"
-                    target="_blank"
-                    rel="noopener noreferrer">training provider
-                    </a>, approved by the DfE
-                </li>
-                <li>if you change your mind about how you want to run your training, contact:
-                    <%= render MailToSupportComponent.new %>
-                </li>
-            </ul>
+      <p class="govuk-body">As soon as possible:</p>
+      <ul class="govuk-list govuk-!-margin-bottom-7t govuk-list--bullet govuk-list--spaced">
+        <% unless appropriate_body_form.body_appointed? %>
+          <li>tell us which appropriate body you’ve appointed for your ECTs</li>
+        <% end %>
+        <li>add the new ECTs and mentors to this service so we can check their eligibility for DfE funding</li>
+        <li>use this service to tell us which mentor has been assigned to each ECT</li>
+      </ul>
 
-        <%- end -%>
+      <p class="govuk-link">
+          <%= govuk_link_to("Continue to manage your training", schools_dashboard_path)  %>
+      </p>
```

Comment:
Not a blocker, just curious why the styled paragraph is needed around the link?

Reply:
I'm not sure. I'll check the prototype and fix it if needed.

Reply:
It should be a `<p class="govuk-body">` I think.